gamemodes/horde/gamemode/perks/warlock/warlock_base.lua

```diff
@@ -62,3 +62,14 @@ PERK.Hooks.Horde_OnUnsetPerk = function(ply, perk)
         ply:SetMaxArmor(100)
     end
 end
+
+PERK.Hooks.Horde_OnPlayerDamage = function ( ply, npc, _, _, dmginfo )
+    if !ply:Horde_GetPerk( "warlock_base" ) then return end
```

Comment:
replace `!` with `not `

Reply:
That would not be correct cause this lua is consistently using ! instead of not